### ISSUE TITLE
Develop and test data extraction script

### ISSUE THREAD

**Issue by FeldmanMike** [FeldmanMike/dc311#3]

**Comment by FeldmanMike** [comment]
URLs to data:
* __2021__: https://opendata.dc.gov/datasets/DCGIS::311-city-service-requests-in-2021
* __2022__: https://opendata.dc.gov/datasets/DCGIS::311-city-service-requests-in-2022
* __2023__: https://opendata.dc.gov/datasets/DCGIS::311-city-service-requests-in-2023
* __2024__: https://opendata.dc.gov/datasets/DCGIS::311-city-service-requests-in-2024


Plan is to extract data from years 2021, 2022, 2023, and 2024.

* 2021 and 2022 will be used for training
* 2023 will be used for validation (e.g., hyperparamater tuning)
* 2024 YTD will be used for a holdout test set (after training on all of 2021, 2022, and 2023)
